news/2018-06-02-mappers-choice-awards-2017-results.md

```diff
@@ -0,0 +1,345 @@
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/whiteout.jpg)](https://osu.ppy.sh/s/653740)
+
+SOUND VOLTEX has always been a loyal customer for both mania mappers and mania players, and this one too isn't an exception when we talking about rythm game category. **[Tofu1222](https://osu.ppy.sh/u/6089608)** manages to bring such an interesting and various LN layering concept into the map that suits with how the song incorporates lot of synth into it. Overall a really intriguing chart with a well deserved spot in this category.
+
+*written by [Maxus](https://osu.ppy.sh/u/4335785)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/swampgator.jpg)](https://osu.ppy.sh/s/613792)
+
+Inspired by **[Wonki](https://osu.ppy.sh/u/7898495)**'s first Zekk map, **[Trigger](https://osu.ppy.sh/s/555076)**, he created another popular mapset featuring quite aggressive SVs and jackhammers. In **[Swampgator](https://osu.ppy.sh/s/613792)**, we can see a development in the variation of SVs used, and how they fit with the patterns and the music in general to make it actually enjoyable to play. With an entry of Spotlight in that month, and quite a noticeable amount of favorites, it's well deserving of an award.
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/hitorigoto.jpg)](https://osu.ppy.sh/s/602861)
+
+Definitely one of the most popular song just by looking at the amount of plays that its standard counterpart has, and it's clearly not the exception for mania either. **[Tofu1222](https://osu.ppy.sh/u/6089608)** manages to bring such an unique touch with the mixing of the piano keysound that further develop the players enjoyment, and not stopping on that, he also uniquely bring his more unorthodox style of pattern approachs in the anime map by putting vocal pitch emphasis into kiai section by layering it into jackhammer. With multiple plus factor that he put into the map, he manages to bring himself into the top spot for the japanese category.
+
+*written by [Maxus](https://osu.ppy.sh/u/4335785)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/dragonforce.jpg)](https://osu.ppy.sh/s/584941)
+
+Being famous for making DragonForce maps, **[Surono](https://osu.ppy.sh/users/3611370)** brought us with a great osu!mania creation of DragonForce marathon in 2017. This map definitely makes up a training for stamina, with jumpstreams of 190 bpms mapped throughout the song, but what's more, its variated patterns in vocal part and guitar solo part marks a noticeable characteristic of this map, thus making it a great fit of Western category.
```

Comment:
brought us with a -> brought to us a
variated -> variety of (just avoid vocab too complex)
in vocal part and guitar solo part marks -> for vocals and guitar solos mark